### DISCUSSION TITLE
Filter API

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#613] [Ideas]
I think that an official and easier way to add filters is to make a Filtering API. As scammers try new tactics to try to, well, scam people, new filter modes are gonna be required.

This would help with the readability and navigation of the code. Of course, on the front-end nothing would change, but under-the-hood would.

_This is just an idea. You can upvote this (not the reaction, as in the arrow) if you like this idea._

**Comment by DinhHuy2010** [comment]
make api in python rights???

**Reply by TechStudent10** [reply]
um yes

**Reply by DinhHuy2010** [reply]
that hard

**Comment by ThioJoe** [comment]
what did you have in mind? You can already search for strings, characters, even regex. Which is pretty much as custom as you can want

**Reply by DinhHuy2010** [reply]
ok

**Reply by TechStudent10** [reply]
As I said above,

> As scammers try new tactics to try to, well, scam people, new filter modes are gonna be required.

Who knows what new tactics scammers could create.
